lib/components/atoms/Video/tests/Video.test.js

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { shallow, render } from 'enzyme';
+
+import { VideoVanilla } from '../index';
+import defaultConfig from '../Video.mock';
+
+const videoContainer = (
+  <VideoVanilla {...defaultConfig}>
+    <source src={defaultConfig.src} />
+    <track />
+  </VideoVanilla>
+);
+describe('<Video />', () => {
```

Comment:
Where are you checking for Default Track and The track you can pass from the body

Reply:
done